pkg/k8s/clientset.go

```diff
@@ -89,6 +94,8 @@ func (c clientSet) Apply(ctx context.Context, obj runtime.Object, namespace stri
 		return errors.Errorf("%s/%s: could not convert runtime object into unstructured data: %w", namespace, name, err)
 	}
 
+	slog.Debug("applying object", "namespace", namespace, "forceConflicts", forceConflicts, "object", lazyObjectLogValue{obj})
```

Comment:
I just used `slog` here without having a broader understanding how Smithy propagates desired log levels. Happy to make changes so it aligns better with how Smithy approaches logging.

Reply:
I think that we might move this internally and remove it from the OSS this week - do you mind if we hold the change for now?

Reply:
Sure, no rush.